src/app/page/Home/VolunteerHome.jsx

```diff
@@ -2,39 +2,68 @@ import { AppBar, Box, Tab, Tabs, Typography } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import PropTypes from "prop-types";
 import React, { useEffect, useState } from "react";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core/styles";
 
 import User from "../../model/User";
 import {
-  getAllAssignedMissions,
+  getAllPlanningMissionss,
   getAllCompletedMissions,
-  getAllStartedMissions,
-  getAllSuggestedMissions,
+  getAllInProgressMissions,
+  getAllProposedMissions,
 } from "./missionHelpers";
 import VolunteerHomeMissionList from "./VolunteerHomeMissionList";
 
+const theme = createMuiTheme({
```

Comment:
If you want to steal the styles from here this should prevent you from having to overwrite the theme in this way.
https://github.com/factn/resilience-app/blob/master/src/app/page/Request/foodbox/FoodboxFlow.jsx#L42-L48

And then we can lift this up into it's own component in another pr